RELEASES.md

```diff
@@ -6,48 +6,68 @@ Below are the versioning schemes used for publishing debs:
+ 8. Push to your test PPA so upload reviewers can install easily test packages easily during review
+ ```bash
+    debuild -S
+    dput  ppa:<YOUR_LAUNCHPAD_USER>/ua-client-uploads ./ubuntu-advantage-tools_*_source.changes
 ```
- ./tools/make-release --series <SERIES> --ppa <PPA_URL_FOR_TEST_UPLOADS>
- # follow printed procedure for dput to test PPA, and tag the released commitish
-```
- 11. Create a merge proposal such as [24.2 PR](https://code.launchpad.net/~chad.smith/ubuntu/+source/ubuntu-advantage-tools/+git/ubuntu-advantage-tools/+merge/385073), [24.3 PR](https://code.launchpad.net/~chad.smith/ubuntu/+source/ubuntu-advantage-tools/+git/ubuntu-advantage-tools/+merge/389745)
- 11. For SRUs: Create an "upload a new version bug" in https://bugs.launchpad.net/ubuntu/+source/ubuntu-advantage-tools/+bugs
+ 9. Create a merge proposal such as [27.0.2 PR](https://code.launchpad.net/~chad.smith/ubuntu/+source/ubuntu-advantage-tools/+git/ubuntu-advantage-tools/+merge/402459) which targets `ubuntu/devel`
+ 10. Add 2 review slots for `canonical-server` and `canonical-server-core-reviewers`
+ 11. **For SRUs:** Create an "upload a new version bug" in https://bugs.launchpad.net/ubuntu/+source/ubuntu-advantage-tools/+bugs like [27.0](https://bugs.launchpad.net/ubuntu/+source/ubuntu-advantage-tools/+bug/1926361) or [20.3](https://bugs.launchpad.net/ubuntu/+source/ubuntu-advantage-tools/+bug/1869980)
  12. Describe the need, provide testing PPA and describe test instructions
- 13. Ping appropriate maintainer about upload and verification
- 14. Validate tests, accept upload and ship it
-
-Previous release bugs:
-| ------- |
-| [20.3 Focal](https://bugs.launchpad.net/ubuntu/+source/ubuntu-advantage-tools/+bug/1869980) |
+ 13. Create a PR for each target series based off the latest `ubuntu/devel` accepted upload
+ ```bash
+ UA_VERSION=27.1
+ SRU_BUG=TODO_FROM_STEP_11
+ rm -rf ../out
+ for release in xenial bionic focal groovy hirsute do
+  git checkout release-${UA_VERSION}-impish -B upload-${UA_VERSION}-$release
+  case "${release}" in
+      xenial) version=${UA_VERSION}~16.04.1;;
+      bionic) version=${UA_VERSION}~18.04.1;;
+      focal) version=${UA_VERSION}~20.04.1;;
+      groovy) version=${UA_VERSION}~20.10.1;;
+      hirsute) version=${UA_VERSION}~21.04.1;;
+  esac
+  dch -v ${version} -D ${release} Backport new upstream release: (LP: #${SRU_BUG}) to $release
+  git commit -m 'changelog backport to ${release}' debian/changelog
+  git push <your_launchpad_username> upload-${UA_VERSION}-$release
+  build-package # to create release-specific dsc files used to upload in step 16
+ ```
+ 14. Create PRs for each SRU target release @ `https://code.launchpad.net/~<YOUR_LP_USER></YOUR_LAUNCHPAD_USER>/ubuntu/+source/ubuntu-advantage-tools/+git/ubuntu-advantage-tools/`. Make sure each PR targets the proper `ubuntu/$release-proposed` branch for each release.
+ 15. Add both `canonical-server` and `canonical-server-core-reviewers` as review slots on the PR.
```

Comment:
> Maybe I'm remembering incorrectly, but shouldn't this be saying that each MP should target the impish branch?

You are correct here. this direction is concorrect, all branches should target release-27-impish. will fix.

> 
> Also, as annoying as it is that every platform uses different terms for PRs, we should try to use the appropriate term per platform, which means "merge proposal'/"MP" here I believe. 
Yeah, and even more broken and incorrect was my own mapping of proposal to the underlying SCM used. I was using PR for "git" based projects and "MP" for bazaar (because launchpad/bazaar were tightly coupled historically in my head).

So MP it is.